src/target/nrf51.c

```diff
@@ -234,10 +234,10 @@ static bool nrf51_mass_erase(target_s *t)
 	if (!nrf51_wait_ready(t, NULL))
 		return false;
 
-	platform_timeout_s timeout = {};
-	platform_timeout_set(&timeout, 500);
+	platform_timeout_s timeout = {0};
```

Comment:
Because of how `platform_timeout_set()` works, this assignment can be dropped entirely